spec/models/elastic_sayt_suggestion_spec.rb

```diff
@@ -1,29 +1,47 @@
 require 'spec_helper'
 
-# Temporarily disabling these specs during ES56 upgrade
-# https://cm-jira.usa.gov/browse/SRCH-821
 describe ElasticSaytSuggestion do
   fixtures :affiliates
   let(:affiliate) { affiliates(:basic_affiliate) }
+  let(:search_params) do
+    {
+      q: 'suggest',
+      affiliate_id: affiliate.id,
+      size: 1,
+      offset: 0,
+      language: affiliate.indexing_locale
+    }
+  end
+  let(:search) { ElasticSaytSuggestion.search_for(search_params) }
 
   before do
     ElasticSaytSuggestion.recreate_index
     affiliate.sayt_suggestions.delete_all
     affiliate.locale = 'en'
   end
 
-  pending ".search_for" do
-    describe "results structure" do
+  describe '.search_for' do
+    describe 'results structure' do
       context 'when there are results' do
+        let(:search_params) do
+          {
+            q: 'suggest',
+            affiliate_id: affiliate.id,
+            size: 1,
+            offset: 1,
+            language: affiliate.indexing_locale
+          }
+        end
+
         before do
           affiliate.sayt_suggestions.create!(phrase: 'hi suggest me', popularity: 30)
           affiliate.sayt_suggestions.create!(phrase: 'suggest me too', popularity: 29)
           affiliate.sayt_suggestions.create!(phrase: 'suggest me three suggests', popularity: 28)
           ElasticSaytSuggestion.commit
         end
 
-        it 'should return results in an easy to access structure ordered by most popular' do
-          search = ElasticSaytSuggestion.search_for(q: 'suggests', affiliate_id: affiliate.id, size: 1, offset: 1, language: affiliate.indexing_locale)
+        it 'returns results in an easy to access structure ordered by most popular' do
+          search = ElasticSaytSuggestion.search_for(search_params)
           expect(search.total).to eq(3)
```

Comment:
Just a note in rails 5 this will be 
          `search = ElasticSaytSuggestion.search_for(params: search_params)`

May want to create a Jira ticket when this goes to pro so I can update it in the rails 5 release branch.

Reply:
Per our discussion, this does not need to change for Rails 5.